modules/ROOT/pages/add-api-alert-task.adoc

```diff
@@ -1,21 +1,27 @@
-= Adding an API Alert
+= Add API Alerts
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-. In API Manager 2.x, in API Administration, select an API instance, and on the API instance dashboard, select Alerts.
-. Click Add Alert and set the alert options:
-* Name: Type an alert name, for example `red-alert`.
-* Enabled: Accept the default, Enabled, for example.
-* Severity: Accept the default, Critical.
-* Alert type: Select Request Count, for example.
-* When number of occurrences are: Select Greater.
-* than: Select 1.
-* For at least: Select an entry in the range 1 - 999999999 to specify the time intervals as a sliding windows.
-* Recipients: Select the names of users in your business group, or type the email addresses, of users who will receive notification when the alert fires. Press RETURN.
+You can add one or more API alerts for your API instances to track and manage APIs.
+
+To add an API alert:
+
+. Navigate to *API Manager* > *API Administration* and click the API instance for which you want to add an alert.
+. From the left navigation select *Alerts* > *Add alert*.
+. Specify the alert options:
+* *Name*: Type an alert name, for example `trial-alert`.
```

Comment:
If ALL the bullet-list entries were just a couple of words, I would say leave them on one line. But they are not, so please follow our standard bullet-list style:
```suggestion
* *Name*
+
Type an alert name, for example `trial-alert`.
```

Reply:
Done! Thanks!